Package.swift

```diff
@@ -128,7 +129,7 @@ let package = Package(
         ),
         .target(
             name: "TuistAutomation",
-            dependencies: ["XcodeProj", "SPMUtility", "TuistCore", "TuistSupport"]
+            dependencies: ["XcodeProj", "SPMUtility", "TuistCore", "TuistSupport", "XcbeautifyLib"]
```

Comment:
Ah nice didn't realise it had a library interface too - that's quite handy to build onto of